hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientMessage.java

```diff
@@ -214,7 +214,7 @@ public int getNumberOfBackupAcks() {
      * @return The ClientMessage with the new dataOffset field value.
      */
     public ClientMessage setNumberOfBackupAcks(final int numberOfAcks) {
```

Comment:
can you also update this interface parameter to `byte`? Or it may be short 'f we want to utilize 0-256 range?